test/spec.js

```diff
@@ -1442,5 +1442,9 @@ e.g. {"body": {"status: "registered"}}`);
 
 		})
 
+        afterEach(() => {
```

Comment:
would you mind trying to remove the `beforeEach(fetchMock.restore)`. Doing it after tests is better. I think doing it before is a throwback to when restore was less error tolerant